src/client/common/cy/tooltips/format-content.js

```diff
@@ -5,54 +5,50 @@ const config = require('../../config');
-  let shortArray = trimValue(revisedList, config.defaultEntryLimit);
+  let shortArray = trim ? trimValue(revisedList, config.defaultEntryLimit):
+                   filterChemicalFormulas(pair[1]);
   let expansionLink = revisedList.length > config.defaultEntryLimit ?
-    h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, 'more »') : h('div.error');
-
+  h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, trimString(trim)) : h('div.error');
   return h('div.fake-paragraph', [
     h('div.field-name', 'Synonyms:'),
     valueToHtml(shortArray, true),
     expansionLink
   ]);
 };
-const nameHandler = (pair, expansionFunction) => {
-  let shortArray = filterChemicalFormulas(pair[1]);
-  return h('div.fake-paragraph', [
-    h('div.field-name', 'Synonyms:'),
-    valueToHtml(shortArray, true),
-    h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, '« less')
-  ]);
-};
 
 //Handle database related fields
-const databaseHandlerTrim = (pair, expansionFunction) => {
-  const expansionLink = h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, 'more »');
+const databaseHandler = (pair, expansionFunction, trim) => {
+  const expansionLink = h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, trimString(trim));
   if (pair[1].length < 1) { return h('div.error'); }
-  return generateDatabaseList(sortByDatabaseId(pair[1]), true, expansionLink);
-};
-const databaseHandler = (pair, expansionFunction) => {
-  const expansionLink = h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, '« less');
-  if (pair[1].length < 1) { return h('div.error'); }
-  return generateDatabaseList(sortByDatabaseId(pair[1]), false, expansionLink);
+  return generateDatabaseList(sortByDatabaseId(pair[1]), trim, expansionLink);
 };
 
 //Handle interaction/Detailed views related fields
-const interactionHandlerTrim =(pair, expansionFunction, title) => {
-  let maxViews=8;
-  const expansionLink = pair[1].length>maxViews? h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, 'more »'):'';
+let maxListEntries=8;
+const viwerListHandler =(pair, expansionFunction, trim, title) => {
+  let db = config.databases;
+  const inner = (database, data, isDBVisble, index) => {
+    let link = db.filter(value => database.toUpperCase() === value.database.toUpperCase());
+    return h('a.db-link' ,{href:'/view?',search: queryString.stringify({
+      uri: link[0].url + link[0].search + data, 
+      title:title, removeInfoMenu:true}),
+    target: '_blank', }, 'Interaction '+(index+1));
+  };
+  const expansionLink = pair[1].length>maxListEntries? h('div.more-link', { onclick: () => expansionFunction(pair[0]) }, trimString(trim)):'';
   if (pair[1].length < 1) { return h('div.error'); }
-  return generateDetailedViewList(sortByDatabaseId(pair[1]), pair[1].length>maxViews, expansionLink,maxViews,title);
+  return interactionList(sortByDatabaseId(pair[1]),trim, expansionLink,maxListEntries,inner);
```

Comment:
can we make the function args consistent i.e make trim always the last param or something like that.

Reply:
I can make it more consistent (trim last almost everywhere) though in the case of line 33 and 103 I think it makes more sense to include title last as it is a parmar that is only used by viwerListHandler so if I were to move trim to last I would need to handle title in all other functions instead of just ignoring it.